lib/bundler/audit/scanner.rb

```diff
@@ -148,8 +148,7 @@ def scan_specs(options={})
 
         @lockfile.specs.each do |gem|
           @database.check_gem(gem) do |advisory|
-            unless (ignore.include?(advisory.cve_id) ||
-                    ignore.include?(advisory.osvdb_id))
+            if advisory.has_supported_identifier?
```

Comment:
What are you trying to do here?

Reply:
I was misunderstanding what the `ignore` set was doing.  Basically I was setting up a scheme where the tool controls what advisory types are allowed, rather than just going off what data was present.  I thought that was what the ignore was doing, however I was quite wrong about that!  All removed now